panel/src/components/Forms/Writer/Toolbar.vue

```diff
@@ -69,7 +69,18 @@ export default {
 			type: Boolean
 		},
 		marks: {
-			default: true,
+			default: () => [
```

Comment:
Without a predefined toolbar layout, the buttons are all over the place when you just pass true or null.